static/js/notifications.js

```diff
@@ -701,14 +696,20 @@ export function reify_message_id(opts) {
 }
 
 export function register_click_handlers() {
-    $("#out-of-view-notification").on("click", ".compose_notification_narrow_by_topic", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        narrow.by_topic(message_id, {trigger: "compose_notification"});
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("#out-of-view-notification").on("click", ".out-of-view-notification-close", (e) => {
-        clear_compose_notifications();
+    $("#above_composebox_banners").on(
+        "click",
+        ".narrow_to_recipient .above_composebox_banner_action_link",
+        (e) => {
+            const message_id = $(e.target).data("message-id");
```

Comment:
Why the change from `currentTarget` to `target` here?

Reply:
No reason, at this point at least. I might have changed this earlier when debugging the link.